tests/unit/test_minion.py

```diff
@@ -427,6 +427,8 @@ def test_when_ping_interval_is_set_the_callback_should_be_added_to_periodic_call
     @slowTest
     def test_when_passed_start_event_grains(self):
         mock_opts = self.get_config("minion", from_scratch=True)
+        # provide mock opts an os grain since we'll look for it later.
```

Comment:
```suggestion
        # provide mock opts and os grain since we'll look for it later.
```

Reply:
"Provide opts an os grain" this is correct.